src/__mocks__/api-handlers.js

```diff
@@ -4,6 +4,7 @@ import {MOCK_COMMENTS, MOCK_ISSUES} from '../utils/GitHub'
 
 const httpOk = 200
 const httpNotFound = 404
+const created = 201
```

Comment:
Please rename this variable to be consistent with the other HTTP status code variables, e.g. `httpCreated`.

Reply:
renamed